### ISSUE TITLE
onAnnotationChanged is not firing when we use the free drawing tool at the first use

### ISSUE THREAD

**Issue by Frekansapp** [PDFTron/pdftron-react-native#209]
Hi,
After embedding pdftron in our react native app, we upload a pdf file and open it. After opening the file, when we want to make a change (in our case drawing a line on the pdf file) by using the free drawing pen (let's abbreviate as FDP).

We select from the top menu, the FDP and draw a random line. When we do this and try to save the file, it does not save the changes made on the file. But if we select another tool from the toolbar and press the save button, then the file is saved with the changes made. We think that there is a problem in saving the file with the changes.
So, to summarize we want the changes saved without pressing another tool's button. 

https://streamable.com/iornnq

 - Platform:Android
 - Device: Xioami Note 8 Pro
 - OS: Android 9.0

**Comment by sgong-pdftron** [comment]
Hi @Frekansapp the FDP is meant to be multi-stroke. If you would like to end the drawing session early, you can call commitTool (https://github.com/PDFTron/pdftron-react-native#committool). Could you give a try?

Thanks.

**Comment by Frekansapp** [comment]
Where is the commit tool ? I couldn't find it.

**Comment by sgong-pdftron** [comment]
Sorry we updated our API page, the new link is here:
https://github.com/PDFTron/pdftron-react-native/blob/master/API.md#committool

**Comment by Frekansapp** [comment]
Before i save called this method this.viewer.setToolMode(Config.Tools.annotationCreateFreeHand);
and then save it. It's working for now. Thank for your answer but coomit tool does not fit my scenario

**Comment by sgong-pdftron** [comment]
@Frekansapp I see, it is strange that commit tool did not work for you, and glad switching tool is working for you instead.
I would recommend using Config.Tools.pan instead (it should serve the same purpose, while go back to panning mode to avoid any further drawing.
